webapp/app/static/css/base.css

```diff
@@ -139,6 +139,7 @@ img.invalid-feedback{
 }
 
 .error-found-line{
+    margin-left: var(--spacing-02);
```

Comment:
Why do we have to add the margin as well?

Reply:
If we have two fields next to each other and the right one has an error, the line is directly next to the left field. Looks .... not nice. :D

Reply:
Thinking about another solution right now, hold on. :D

Reply:
Alright, just pushed another solution. (basically what bootstrap is doing with their columns - I have to say, I like it. How about you?